streamer/src/nonblocking/quic.rs

```diff
@@ -1083,27 +1093,35 @@ async fn handle_connection(
                 sleep(throttle_duration).await;
             }
         }
-        stream_load_ema.increment_load(params.peer_type);
+        stream_load_ema.increment_load(peer_type);
         stream_counter.stream_count.fetch_add(1, Ordering::Relaxed);
         stats.total_streams.fetch_add(1, Ordering::Relaxed);
         stats.total_new_streams.fetch_add(1, Ordering::Relaxed);
-        let packet_sender = &params.packet_sender;
-        let mut maybe_batch = None;
+
+        let mut meta = Meta::default();
+        meta.set_socket_addr(&remote_addr);
+        meta.set_from_staked_node(matches!(peer_type, ConnectionPeerType::Staked(_)));
+        let mut accum = PacketAccumulator {
```

Comment:
maybe just add `new` to this `PacketAccumulator`?

Reply:
done